doc/report.md

```diff
@@ -13,59 +13,126 @@ delle mutazioni sia dei conigli che dell'ambiente circostante.
+
+| x | F  | f | 
+| :--: | :--: | :--: |
+| f | fF  | ff | 
+| f | fF  | ff | 
+
+* Quando viene introdotta una mutazione nel 50%+1 delle coppie formate, solamente un figlio su quattro la presenterà; verrà mostrato per quel gene una coppia di alleli dovuta dalla mutazione e che prescinde dagli alleli dei genitori e dalle combinazioni previste dal [Quadrato di Punnett](https://it.wikipedia.org/wiki/Quadrato_di_Punnett). La mutazione viene introdotta come **omozigote**, ovvero con due alleli di tipo mutato, a prescindere che essi siano dominanti o recessivi. </br> Ad esempio, l'utente sceglie di inserire la mutazione dominante per il gene riguardante il colore della pelliccia (lettera <tt>f</tt>), perciò tutti i coniglietti con fenotipo di base presenti fino a quel momento devono essere formati da due alleli recessivi (<tt>ff</tt>), mentre il coniglietto mutato sarà formato da due allei dominanti (<tt>FF</tt>). </br> Di seguito è riportato un esempio dei quattro figli generati da un coppia in cui viene introdotta questa mutazione.
+
+| x | f  | f | 
+| :--: | :--: | :--: |
+| f | ff  | *FF* | 
+| f | ff  | ff | 
+
+* Nel caso in cui siano definite più mutazioni (su vari geni) durante la medesima generazione si cercherà di mantenere un'unica mutazione per coniglio, le mutazioni si andranno ad accumulare sullo stesso figlio solo nel caso in cui non si disponga di abbastanza conigli.
+
+### Requisiti Utente
+Di seguito sono riportati i requisiti visti nell'ottica di cosa può fare l'utente con l'applicativo.
+* L'utente visualizzerà lo stato attuale della simulazione nel pannello principale: quanti conigli sono presenti e quali caratteristiche possiedono, le condizioni ambientali e l'avanzamento temporale delle generazioni;
+* Tale pannello sarà realizzato con un'interfaccia grafica 2D accattivante ed intuitiva.
+* L'utente potrà modificare il clima in cui si riproducono i coniglietti.
+* L'utente potrà scegliere quali mutazioni introdurre, specificando quali di esse sono dominanti e quali recessive.
+* L'utente potrà aggiungere vari fattori che condizioneranno l'evoluzione.
+* L'utente potrà visualizzare il patrimonio genetico di ogni coniglietto, avendo ogni gene rappresentato da una lettere che compare maiscuola se ne rappresenta l'allele dominante e minuscola se ne rappresenta l'allele recessivo.
```

Comment:
```suggestion
* L'utente potrà visualizzare il patrimonio genetico di ogni coniglietto, avendo ogni gene rappresentato con una coppia di caratteri della stessa lettera dell'alfabeto, questa se compare maiuscola allora il relativo allele è dominante mentre se è minuscola è recessivo.
```

Reply:
In realtà non mi piace molto neanche la mia versione, in entrambi i casi, secondo me, non si capisce bene cosa intendiamo